wp-content/themes/pub/wporg-learn-2024/inc/head.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+add_action( 'init', __NAMESPACE__ . '\init' );
+
+/**
+ * Handles adding/removing hooks as needed.
+ */
+function init() {
+	add_filter( 'document_title_parts', __NAMESPACE__ . '\document_title' );
+	add_filter( 'document_title_separator', __NAMESPACE__ . '\document_title_separator' );
+}
+
+/**
+ * Filters document title to add context based on what is being viewed.
+ *
+ * @param array $parts The document title parts.
+ * @return array The document title parts.
+ */
+function document_title( $parts ) {
+	global $wp_query;
+
+	$parts['site']  = __( 'Learn WordPress', 'wporg-learn' );
```

Comment:
It seems we usually finish with the domain, eg.

| Main | Dev Resources | Patterns |
|--------|--------|--------|
| ![Screenshot 2024-07-15 at 4 19 47 PM](https://github.com/user-attachments/assets/49af3729-9a93-4e4b-888f-ed0f7ae3b5c9) | ![Screenshot 2024-07-15 at 4 16 13 PM](https://github.com/user-attachments/assets/b4967c2b-ad09-4a89-8013-cd85083bf72d) | ![Screenshot 2024-07-15 at 4 17 17 PM](https://github.com/user-attachments/assets/b74592f9-3b12-4fbf-aefe-4c092e55b616) | 

```suggestion
	$parts['site']  = __( 'Learn.WordPress.org', 'wporg-learn' );
```

Reply:
I originally used Learn.WordPress.org, but I decided to follow the ticket description first. @kathrynwp, can you help make the final confirmation for this, thanks.

Changed back first here 7ad5444